src/main/java/bio/terra/cloudfiletodatastore/FileCreatedMessageHarness.java

```diff
@@ -0,0 +1,14 @@
+package bio.terra.cloudfiletodatastore;
```

Comment:
Instead of a contrived event type like `GCSEvent` or generic `T`, we should probably just stick with the `v1/storage` specification `StorageObjectData` and the wrapper class `StorageObjectEventMessage` to handle the conversion directly. We can adapt to different cloud providers in the future with separate wrapper classes that handles provider-specific conventions. The main theme here is to make `CloudEventsHarness` adaptable across cloud providers and event types.